src/index.njk

```diff
@@ -26,6 +26,30 @@ masthead: true
         <p class="govuk-body govuk-!-margin-bottom-0"><a href="/patterns/" class="govuk-link govuk-!-font-weight-bold">Browse patterns</a></p>
       </div>
 
+      <div class="govuk-grid-column-full">
+        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl">
+      </div>
+
+      <div class="govuk-grid-column-two-thirds govuk-!-mb-r4">
+        <h2 class="govuk-heading-l">Principles we follow</h2>
+        <p class="govuk-body">
+The GOV.UK Design System helps service teams follow the:
+<ul>
+  <li><a class="govuk-link" href="https://www.gov.uk/guidance/government-design-principles">Government Design Principles</a></li>
+  <li><a class="govuk-link" href="https://www.gov.uk/service-manual">GOV.UK Service Manual</a></li>
+</ul>
+
+This website also follows Government Digital Service (GDS) guidance for:
+<ul>
+  <li><a class="govuk-link" href="https://www.gov.uk/guidance/style-guide">style, spelling and grammar</a></li>
+  <li><a class="govuk-link" href="https://www.gov.uk/guidance/content-design">content design</a></li>
+</ul>
+        </p>
```

Comment:
Argh, need to style correctly — will have a poke at other pages to see what to do. Still not sure on the bullet approach just yet btw.